runtime/compiler/env/j9method.cpp

```diff
@@ -6504,8 +6505,9 @@ TR_ResolvedJ9Method::newInstancePrototypeSignature(TR_Memory * m, TR_AllocationK
    TR_ASSERT(_j9classForNewInstance, "Must have the class for newInstance");
    J9Class * clazz = _j9classForNewInstance; //((J9Class*)((uintptr_t)(ramMethod()->extra) & ~J9_STARTPC_NOT_TRANSLATED);
    char    * className = fej9()->getClassNameChars(_fe->convertClassPtrToClassOffset(clazz), clen);
-   char    * s = (char *)m->allocateMemory(clen+nameLength()+signatureLength()+3, allocKind);
-   sprintf(s, "%.*s.%.*s%.*s", clen, className, nameLength(), nameChars(), signatureLength(), signatureChars());
+   size_t   len = clen+nameLength()+signatureLength()+3;
```

Comment:
Some spaces would be welcome here:
```c
   size_t    len = clen + nameLength() + signatureLength() + 3;
```
(including an extra one before the name for alignment).